src/components/capture/RediscoverButton.tsx

```diff
@@ -1,21 +1,18 @@
 import { Button } from '@mui/material';
 import useDiscoverCapture from 'components/capture/useDiscoverCapture';
+import useEntityWorkflowHelpers from 'components/shared/Entity/hooks/useEntityWorkflowHelpers';
 import { FormattedMessage } from 'react-intl';
 import { Entity } from 'types';
 
 interface Props {
     entityType: Entity;
     disabled: boolean;
-    callFailed: Function;
     postGenerateMutate: Function;
 }
 
-function RediscoverButton({
-    entityType,
-    disabled,
-    callFailed,
-    postGenerateMutate,
-}: Props) {
+function RediscoverButton({ entityType, disabled, postGenerateMutate }: Props) {
+    const { callFailed } = useEntityWorkflowHelpers();
```

Comment:
We fetch this same function for both here and the `generate` button. We should just have the `useDiscoverCapture` hook pull this out and not have it passed.